teachers.rb

```diff
@@ -0,0 +1,40 @@
+class Teachers 
```

Comment:
Any reason for the choice to call this calls `Teachers`? This class represents a parent class to the `ApprenticeTeacher` and `SeniorTeacher` classes. Ideally the code should allow the possibility of creating a new `Teacher`, in which case it would be grammatically awkward to say `Teachers.new` to get a new instance of the class in the same way the code would call `ApprenticeTeacher.new`.